api/src/main/java/com/epam/pipeline/manager/cluster/costs/ClusterCostsMonitoringServiceCore.java

```diff
@@ -28,29 +31,46 @@
 
 import java.math.BigDecimal;
 import java.math.RoundingMode;
+import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
-import java.util.function.Function;
-import java.util.stream.Collectors;
 
 @Service
 @RequiredArgsConstructor
 @Slf4j
 public class ClusterCostsMonitoringServiceCore {
     private static final int DIVIDE_SCALE = 5;
     private static final int MINUTES_IN_HOUR = 60;
-
+    private static final int DEFAULT_PAGE = 1;
+    private static final int DEFAULT_PAGE_SIZE = 20;
     private final PipelineRunCRUDService pipelineRunCRUDService;
     private final PipelineRunManager pipelineRunManager;
 
     @SchedulerLock(name = "ClusterCostsMonitoringService_monitor", lockAtMostForString = "PT10M")
     public void monitor() {
         log.debug("Started cluster costs monitoring");
-        final Map<Long, PipelineRun> masters = ListUtils.emptyIfNull(pipelineRunManager
-                .loadRunningPipelineRuns()).stream()
-                .filter(PipelineRun::isMasterRun)
-                .collect(Collectors.toMap(PipelineRun::getId, Function.identity()));
+        final PagingRunFilterVO filter = new PagingRunFilterVO();
+        filter.setPageSize(DEFAULT_PAGE_SIZE);
+        filter.setStatuses(Collections.singletonList(TaskStatus.RUNNING));
+        filter.setEagerGrouping(false);
+        filter.setUserModified(false);
+        int page = DEFAULT_PAGE;
+        final Map<Long, PipelineRun> masters = new HashMap<>();
+        Integer totalCount = null;
```

Comment:
it seems that there is no need for such var, just use `listPagedResult.getTotalCount()` in line 71

Reply:
Paging was removed according to searchPipelineRuns update with checkMaxPageSize